.github/workflows/process_new_pull_request.yml

```diff
@@ -0,0 +1,230 @@
+name: Assign Reviewer For Contributor
+
+on:
+  pull_request_target:
+    types: [opened]
+
+permissions: read-all
+
+jobs:
+  manage_reviewers:
+    name: Manage Reviewers and Notifications
+    runs-on: ubuntu-24.04
+    permissions:
+      pull-requests: write
+      issues: write
+    steps:
+      - name: Handle Reviewers and Notifications
+        uses: actions/github-script@v7
+        with:
+          script: |
+            // ======================
+            // INPUT SANITIZATION
+            // ======================
+            const validateGitHubUsername = (username) => {
+              if (typeof username !== 'string') return false;
+              return /^[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}$/i.test(username);
+            };
+            
+            const sanitizeUsername = (input) => {
+              return validateGitHubUsername(input) ? input : '';
+            };
+            
+            const sanitizeRepo = (input) => {
+              const allowed = /^[A-Za-z0-9_.-]{1,100}$/;
+              if (typeof input !== 'string') return '';
+              return allowed.test(input) ? input : '';
+            };
+            
+            const sanitizePrNumber = (num) => {
+              const n = Number(num);
+              return Number.isInteger(n) && n > 0 ? n : NaN;
+            };
+            
+            // Core PR metadata with validation
+            if (!context.payload?.pull_request || !context.payload.repository) {
+              core.setFailed('Invalid event payload');
+              return;
+            }
+            
+            const prNumber = sanitizePrNumber(context.payload.pull_request.number);
+            if (isNaN(prNumber)) {
+              core.setFailed(`Invalid PR number: ${context.payload.pull_request.number || 'undefined'}`);
+              return;
+            }
+            
+            const repo = sanitizeRepo(context.payload.repository.name);
+            const owner = sanitizeUsername(context.payload.repository.owner.login);
+            if (!repo || !owner) {
+              core.setFailed(`Invalid repo (${repo}) or owner (${owner})`);
+              return;
+            }
```

Comment:
Why are you including `repo` and `owner` in the error message? If one is invalid, it will always be an empty string, so including its value in the error message isn't useful